fakeredis.py

```diff
@@ -2231,11 +2231,13 @@ class FakePubSub(object):
     PATTERN_MESSAGE_TYPES = ['psubscribe', 'punsubscribe']
     LISTEN_DELAY = 0.1          # delay between listen loops (seconds)
 
-    def __init__(self, decode_responses=False, *args, **kwargs):
+    def __init__(self, connected=True, decode_responses=False, *args, **kwargs):
```

Comment:
I would rather put `connected` after `decode_responses`, so that it doesn't break existing code that passes decode_responses as a positional parameter.